ci/container/Dockerfile

```diff
@@ -55,6 +55,25 @@ RUN export DEBIAN_FRONTEND=noninteractive && \
     cd .. && \
     rm -rf e2fsdroid
 
+# Install an older libtinfo for rules_haskell
+RUN mkdir libtinfo && \
+    cd libtinfo && \
+    wget "http://security.ubuntu.com/ubuntu/pool/universe/n/ncurses/libtinfo5_6.4-2ubuntu0.1_amd64.deb" && \
+    apt -yqq install ./libtinfo5_6.4-2ubuntu0.1_amd64.deb && \
+    cd .. && \
+    rm -rf libtinfo
+
+# Build libunwind from source. The packaged version breaks linking
+RUN apt -yqq install libtool && \
```

Comment:
Could you install this in the exiting layers we have with `apt install`? Please also remove kubic libcontainers apt repo as we don't need it anymore. Podman is available from Ubuntu's apt repo.